code/clue.js

```diff
@@ -48,12 +192,47 @@ const pickKiller = () => {
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerImage').src = mystery.killer.image;
+  document.getElementById('killerAge').innerHTML = (`${mystery.killer.age} years old`)
```

Comment:
This will work aswell!
```suggestion
  document.getElementById('killerAge').innerHTML = `${mystery.killer.age} years old`
```